### ISSUE TITLE
After All goes Well app-debug.apk not sending any HTTP Request to my DNS.

### ISSUE THREAD

**Issue by srom23** [BenChaliah/Arbitrium-RAT#3]
I have done everything but after the "cordova build android" apk installed in Device its **not Sending Any HTTP Request** to my server  After Authrize and connect its shows connected But no Requests _i Have Confirmed  this by simple tool Packet capture app._

**Comment by BenChaliah** [comment]
Did you build the binaries (runFrame & toolbox) and put them to assets (serverAPI) ?

**Comment by srom23** [comment]
Yes i did But it should still Send a First Get Requset. (But no Requsets Got on screen -r "main.py")

**Comment by BenChaliah** [comment]
Can you please email me `ayoub1benchaliah@gmail.com`, I'll check with you in details to see if it's a real issue.
